spinn_front_end_common/interface/provenance/.gitignore

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2021 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.sqlite3
```

Comment:
Any reason to not just put this in the main `.gitignore`? We're not likely to be wanting to commit any databases any time soon…

Reply:
Oh, I see there's one just below. Why? Just so it can run as a demo?

Reply:
Seems to be exactly that. OK, I'll allow it.